client/app/reducers/test.reducer.js

```diff
@@ -0,0 +1,13 @@
+const initialState = []
```

Comment:
so would we call this reducers test `test.reducer.spec.js` ?

Reply:
Not entirely sure just yet